app/submissions/views.py

```diff
@@ -1,9 +1,32 @@
 from rest_framework import viewsets
-
+from rest_framework import generics
 from .models import Submission
 from .serializers import SubmissionSerializer
+from django.http import HttpResponse
+from wsgiref.util import FileWrapper
+from django.shortcuts import get_object_or_404
 
 
 class SubmissionViewList(viewsets.ModelViewSet):
     serializer_class = SubmissionSerializer
     queryset = Submission.objects.all()
+
+
+class FileDownloadListAPIView(generics.ListAPIView):
```

Comment:
Actually, I don't believe it won't l be able to handle the traffic load, that's expected.
I suggest to rewrite this part in Java...

Reply:
![image](https://user-images.githubusercontent.com/34000503/103107192-9830fd80-4644-11eb-9f47-2132103dd68f.png)